Userland/Services/WebDriver/Client.h

```diff
@@ -91,6 +91,7 @@ class Client final : public Web::WebDriver::Client {
     virtual Web::WebDriver::Response take_screenshot(Web::WebDriver::Parameters parameters, JsonValue payload) override;
     virtual Web::WebDriver::Response take_element_screenshot(Web::WebDriver::Parameters parameters, JsonValue payload) override;
     virtual Web::WebDriver::Response print_page(Web::WebDriver::Parameters parameters, JsonValue payload) override;
+    virtual Web::WebDriver::Response get_computed_role(Web::WebDriver::Parameters parameters, JsonValue payload) override;
```

Comment:
Almost there, just this one remaining line should be moved just below `is_element_enabled` above